build.gradle

```diff
@@ -49,62 +49,30 @@ bootJar {
     }
 }
 
-bootBuildImage.doFirst {
-    // check data
-    assert System.getenv("DOCKER_USERNAME") != null
-    assert System.getenv("DOCKER_TOKEN") != null
-}
-
-bootBuildImage {
-    // prepare data
-    def tagLatest = Boolean.valueOf(System.getenv("TAG_LATEST"))
-    def dockerImageName = System.getenv("DOCKER_IMAGE_NAME")
-    def dockerUsername = System.getenv("DOCKER_USERNAME")
-    def dockerToken = System.getenv("DOCKER_TOKEN")
-
-    if (dockerImageName == null) {
-        dockerImageName = "${dockerUsername}/halo"
-    }
-    if (!tagLatest) {
-        dockerImageName = "${dockerImageName}:${project.version}"
-    }
-
-    // config plugin
-    imageName = "${dockerImageName}"
-    docker {
-        publishRegistry {
-            username = "${dockerUsername}"
-            password = "${dockerToken}"
-            email = "hi@halo.run"
-        }
-    }
-}
-
 ext {
-    guavaVersion= "31.0-jre"
-    upyunSdkVersion = "4.2.0"
-    qiniuSdkVersion = "7.2.29"
-    aliyunSdkVersion = "3.11.3"
-    baiduSdkVersion = "0.10.36"
-    qcloudSdkVersion = "5.6.25"
-    minioSdkVersion = "7.1.4"
+    guavaVersion= '31.0.1-jre'
+    upyunSdkVersion = '4.2.3'
+    qiniuSdkVersion = '7.8.0'
+    aliyunSdkVersion = '3.13.2'
+    baiduSdkVersion = '0.10.185'
+    qcloudSdkVersion = '5.6.61'
+    minioSdkVersion = '8.3.4'
     swaggerVersion = "3.0.0"
     commonsFileUploadVersion = "1.4"
-    commonsLangVersion = "3.10"
-    httpclientVersion = "4.5.12"
-    dataformatYamlVersion = "2.11.0"
+    commonsLangVersion = '3.12.0'
+    httpclientVersion = '4.5.13'
     jgitVersion = "5.9.0.202009080501-r"
     flexmarkVersion = "0.62.2"
-    thumbnailatorVersion = "0.4.13"
+    thumbnailatorVersion = '0.4.14'
     image4jVersion = "0.7zensight1"
-    flywayVersion = "7.5.1"
-    h2Version = "1.4.197"
+    flywayVersion = "7.15.0"
+    h2Version = "1.4.199"
```

Comment:
Why do I bump h2 to 1.4.199? Cuz there is a critical issue in 1.4.200, please see https://github.com/h2database/h2database/issues/2078.